File: MonitoriaAgenda_API/Models/Horario.cs
using System.ComponentModel.DataAnnotations;
using System;

namespace api.Models
{
    public class Horario
    {
        [Key]
        public int IdHorario { get; set; }
        public int DiaDaSemana { get; set; }
        public DateTime HoraInicio { get; set; }
        public DateTime HoraFim { get; set; }
        public int IdMonitor { get; set; }
    }
}

File: monitoria-agenda-app/src/store/reducers/horario.js
const INITIAL_STATE = {
    horario: {
        idHorario:0,
        diaDaSemana:0,
        horaInicio:"0",
        horaFim:"0",
        idMonitor:0,
    },
}
export default function setHorario(state = INITIAL_STATE, action) {
    console.log(action)
    if(action.type === 'SET_HORARIO') {
        return {
            ...state,
            horario:action.horario,
        };
    }
    return state;

}